scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc

```diff
@@ -7,13 +7,10 @@ toc::[]
 
 Consider the following tested object maximums when you plan your {product-title} cluster.
 
-These guidelines are based on the largest possible cluster. For smaller
-clusters, the maximums are lower. There are many factors that
-influence the stated thresholds, including the etcd version or storage data
-format.
+These guidelines are based on the largest possible cluster. For smaller clusters, the maximums are lower. There are many factors that
+influence the stated thresholds, including the etcd version or storage data format.
```

Comment:
did you intend to unwrap this line?

Reply:
yes, but looks like there's an extra space that's throwing off the flow; fixing that now! Thanks!